SPECS-SIGNED/fwctl-signed/fwctl-signed.spec

```diff
@@ -0,0 +1,142 @@
+#
+# Copyright (c) 2024 Nvidia Inc. All rights reserved.
+#
+# This software is available to you under a choice of one of two
+# licenses.  You may choose to be licensed under the terms of the GNU
+# General Public License (GPL) Version 2, available from the file
+# COPYING in the main directory of this source tree, or the
+# OpenIB.org BSD license below:
+#
+#     Redistribution and use in source and binary forms, with or
+#     without modification, are permitted provided that the following
+#     conditions are met:
+#
+#      - Redistributions of source code must retain the above
+#        copyright notice, this list of conditions and the following
+#        disclaimer.
+#
+#      - Redistributions in binary form must reproduce the above
+#        copyright notice, this list of conditions and the following
+#        disclaimer in the documentation and/or other materials
+#        provided with the distribution.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+# NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS
+# BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN
+# ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+# CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+#
+
+%{!?_name: %define _name fwctl}
+%{!?_version: %define _version 24.10}
+%{!?_release: %define _release OFED.24.10.0.6.7.1}
+
+%global target_kernel_version_full %(/bin/rpm -q --queryformat '%{RPMTAG_VERSION}-%{RPMTAG_RELEASE}' $(/bin/rpm -q --whatprovides kernel-headers))
+%global target_azurelinux_build_kernel_version %(/bin/rpm -q --queryformat '%{RPMTAG_VERSION}' $(/bin/rpm -q --whatprovides kernel-headers))
+%global target_kernel_release %(/bin/rpm -q --queryformat '%{RPMTAG_RELEASE}' $(/bin/rpm -q --whatprovides kernel-headers) | /bin/cut -d . -f 1)
+
+%global KVERSION %{target_kernel_version_full}
+%global K_SRC /lib/modules/%{target_kernel_version_full}/build
+
+# KMP is disabled by default
+%{!?KMP: %global KMP 0}
+
+%{!?KVERSION: %global KVERSION %{target_kernel_version_full}}
+%global kernel_version %{KVERSION}
+%global krelver %(echo -n %{KVERSION} | sed -e 's/-/_/g')
+# take path to kernel sources if provided, otherwise look in default location (for non KMP rpms).
+%{!?K_SRC: %global K_SRC /lib/modules/%{KVERSION}/build}
+
+# define release version
+%{!?src_release: %global src_release %{_release}_%{krelver}}
+%if "%{KMP}" != "1"
+%global _release1 %{src_release}
+%else
+%global _release1 %{_release}
+%endif
+%global _kmp_rel %{_release1}%{?_kmp_build_num}%{?_dist}
+
+Summary:	 %{_name} Driver
+Name:		 %{_name}
+Version:	 %{_version}
+Release:	 1%{?dist}
+License:	 GPLv2
```

Comment:
Just to be sure, the SPEC file itself has it's own license, and I believe the header comment in this file says we have a choice between GPLv2 OR BSD.  However, I want to be sure that doesn't extend to the fwctl source itself.  If so, then the License: section should explicitly say GPLv2 OR BSD